toolbox

```diff
@@ -477,6 +478,105 @@ list_containers()
 )
 
 
+run_rm()
+(
+    force_remove=$1
+    command_to_run=$2
+    id_or_name=$3
+    keyword=$4
+
+    if $force_remove; then
+        if ! rm_output=$($prefix_sudo podman $command_to_run $id_or_name --force 2>&42); then
+            echo "$0: failed to remove $id_or_name"
+            exit 1
+        fi
+    else
+        if ! rm_output=$($prefix_sudo podman $command_to_run $id_or_name 2>&42); then
+            echo "$0: failed to remove $id_or_name"
+            exit 1
+        fi
+    fi
+    if ! [ "$rm_output" = "" ]; then
+        echo -e "${LRC}$keyword $rm_output removed${NC}"
```

Comment:
Umm... I think that it's better to stay silent unless there has been an error, or the command explicitly demands some output (eg., `list`) or if it's a long running command that needs some progress indicator.

It seems like we need to find the right balance between the classical UNIX commands like `cp(1)`, `ls(1)`, `rm(1)`, etc. which tend to be more terse and things like `git(1)`, `podman(1)`, etc., which are more chatty. So far, we have been following the former, and we can revisit this if it turns out to be too unfriendly. But for the time being let's stay consistent.

Reply:
Done.